CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ description: |-
 
 # VKCS Provider's changelog
 #### v0.5.1 (unreleased)
+- Support root password in vkcs_db_cluster resource
```

Comment:
IMHO, we should place more precise message here because there were corresponding attributes but they were not properly processed, so it's actually a bugfix. And it's related to both cluster and cluster_with_shards.

Reply:
Rewrote